src/components/content-tab-assistant.tsx

```diff
@@ -239,6 +246,7 @@ const AuthenticatedView = memo(
 									transition={ { duration: 0.3 } }
 									initial="initial"
```

Comment:
Let's add here the condition if it should animate or not.
I think this is the only change that is required to fix the issue.

```suggestion
									initial={ ! isInitialRenderRef.current ? 'initial' : 'animate' }
```

Reply:
Very nice! Thanks!

I have just used

```
initial={ isInitialRenderRef.current ? 'animate' : 'initial' }` 
```

instead of

```
initial={ ! isInitialRenderRef.current ? 'initial' : 'animate' }
```